aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -307,48 +308,19 @@ module aptos_framework::account {
         rotation_capability_sig_bytes: vector<u8>,
         account_public_key_bytes: vector<u8>,
         recipient_address: address,
-    ) acquires Account {
-        let addr = signer::address_of(account);
-        assert!(exists_at(addr) && exists_at(recipient_address), error::not_found(EACCOUNT_DOES_NOT_EXIST));
-
-        let pubkey = ed25519::new_unvalidated_public_key_from_bytes(account_public_key_bytes);
-        let rotation_capability_sig = ed25519::new_signature_from_bytes(rotation_capability_sig_bytes);
-
-        // Get the current authentication key of the account and verify that it matches with `account_public_key_bytes`
-        let account_resource = borrow_global_mut<Account>(addr);
-        let auth_key = ed25519::unvalidated_public_key_to_authentication_key(&pubkey);
-        assert!(account_resource.authentication_key == auth_key, error::invalid_argument(EWRONG_CURRENT_PUBLIC_KEY));
-
-        //  Construct a RotationCapabilityOfferProofChallenge struct
-        let rotation_capability_offer_proof_challenge = RotationCapabilityOfferProofChallenge {
-            sequence_number: account_resource.sequence_number,
-            recipient_address,
-        };
-
-        // Verify a digital-signature-based capability that assures us this rotation capability offer was intended by the account owner
-        assert!(ed25519::signature_verify_strict_t(&rotation_capability_sig, &pubkey, rotation_capability_offer_proof_challenge), error::invalid_argument(EINVALID_PROOF_OF_KNOWLEDGE));
-
-        // Add the recipient's address in account owner's rotation capability offer once we verify that this action is intended by the account owner
-        option::fill(&mut account_resource.rotation_capability_offer.for, recipient_address);
+    ) {
+        let _account = account;
```

Comment:
yea, I was thinking that it'd be a breaking api change that way? 

if someone was using this api and they were doing something like 
`offer_rotation_capability_ed25519(account = current_account, rotation_capability_sig_bytes = signature, ...`, it would break when calling the function rather than when it reaches the `abort` statement? 

I wasn't sure if changing the parameter names counts as a breaking api change

Reply:
oh nvm david is deleting these functions in his pr so I'll close this pr